### PR TITLE
Graduate JobPodReplacementPolicy to beta

### PR DESCRIPTION
**Pull request by alculquicondor**
Part of kubernetes/enhancements#3939

### CODE REVIEW

File: content/en/docs/concepts/workloads/controllers/job.md [hunk 1]
```diff
@@ -953,11 +953,12 @@ scaling an indexed Job, such as MPI, Horovord, Ray, and PyTorch training jobs.
...
 ### Delayed creation of replacement pods {#pod-replacement-policy}
 
-{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.29" state="beta" >}}
 
 {{< note >}}
 You can only set `podReplacementPolicy` on Jobs if you enable the `JobPodReplacementPolicy`
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+(enabled by default)
```
Comment (tengqm) [inline content/en/docs/concepts/workloads/controllers/job.md:961]:
```suggestion
(enabled by default).
```

**Review by tengqm — COMMENTED** [review]
This should target dev-1.29 branch, IMO.

**Comment by sftim** [comment]
Oops
/sig apps
/remove-sig storage

File: content/en/docs/concepts/workloads/controllers/job.md (hunk 1 above)
Comment (alculquicondor) [reply content/en/docs/concepts/workloads/controllers/job.md]: Done

**Comment by alculquicondor** [comment]
/hold cancel

I changed the branch long time ago :)

**Pull request merged**